public/static/locales/th/schedule.json

```diff
@@ -62,4 +62,4 @@
       }
     }
   }
-}
+}
```

Comment:
Please stop removing ending new lines. I don’t think that’s the standard for literally any file we use.